### DISCUSSION TITLE
[Projects Beta] Nice to have the ability to assign users to 'draft' issues without needing to create real issues

### DISCUSSION THREAD

**Discussion by creasty** [community/community#5837] [Projects and Issues]
## Why

When you know there's a task that you don't know exactly what it is but do know who has the details, you'd like to create a draft issue and assign it to them; As they have the info you don't, it's reasonable to make it the assignee's responsibility to file a solid and real issue.

<img width="1047" src="https://user-images.githubusercontent.com/1695538/134952504-3d2564a4-4d9d-4005-a952-432d97bced45.png">

**Comment by creasty** [comment]
You can't also use with the `assignee:` filter because of this.

<img width="463" alt="Screen Shot 2021-10-11 at 17 29 25" src="https://user-images.githubusercontent.com/1695538/136758469-08f27eb9-21f0-4864-ad78-51c3f0bb922b.png">

**Comment by Totktonada** [comment]
It seems, now it is tracked in the https://github.com/github/roadmap/issues/285 roadmap issue:

> ## Summary
>
> Allowing draft issues to function as full issues, for example the ability to assign and discuss - both of which are not possible today.

It is in the 'Future' milestone ATM.
